modules/proxistoreBidAdapter.js

```diff
@@ -59,17 +72,28 @@ function _createBidResponse(response) {
     vastUrl: response.vastUrl,
     vastXml: response.vastXml,
     dealId: response.dealId
-  }
+  };
 }
-
 /**
  * Determines whether or not the given bid request is valid.
  *
  * @param bid  The bid params to validate.
  * @return boolean True if this is a valid bid, and false otherwise.
  */
+
 function isBidRequestValid(bid) {
-  return !!(bid.params.website && bid.params.language);
+  const hasNoAd = function() {
+    const pxNoAds = storage.getDataFromLocalStorage(`PX_NoAds_${bid.params.website}`);
```

Comment:
This function call can throw an exception if user has disabled browser to store any data.  

Either wrap it in `try-catch` or check if the `localStorage` is enabled by executing:

```
storage.hasLocalStorage();
```